### ISSUE TITLE
Add feature to allow the user to change avatar

### ISSUE THREAD

**Issue by arundhati24** [fossasia/susi_android#1691]
**Actual Behaviour**

Currently, the Gravatar service is being used to display the avatar of the user.

**Expected Behaviour**

Add feature to allow the user to display one of the following :
* Gravatar
* Avatar uploaded by the user
* Default avatar

**Would you like to work on the issue?**

Yes

**Comment by shikhart98** [comment]
Are you still working on it?

**Comment by arundhati24** [comment]
@shikhart98 Yes, I would be working on it this week. Please check out some of the other open issues.

**Comment by Bhavnaharitsa** [comment]
@arundhati24 Are you still working on this issue and could you tell me what is gravatar ?

**Comment by iamareebjamal** [comment]
http://lmgtfy.com/?q=gravatar

**Comment by Bhavnaharitsa** [comment]
Thank you !
